.github/workflows/publish-binaries.yml

```diff
@@ -66,6 +66,45 @@ jobs:
         file: target/release/${{ matrix.artifact_name }}
         asset_name: ${{ matrix.asset_name }}
         tag: ${{ github.ref }}
+        
+  publish-macos-apple-silicon:
+    name: Publish binary for macOS silicon
+    runs-on: ${{ matrix.os }}
+    needs: check-version
+    continue-on-error: false
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - build: aarch64
```

Comment:
Where `build` is used?

Reply:
Oupsi, seems like a copy-pasta mistake :D 
Removed!